src/cdk-experimental/scrolling/virtual-for-of.ts

```diff
@@ -147,8 +147,15 @@ export class CdkVirtualForOf<T> implements CollectionViewer, DoCheck, OnDestroy
       private _differs: IterableDiffers,
       /** The virtual scrolling viewport that these items are being rendered in. */
       @SkipSelf() private _viewport: CdkVirtualScrollViewport) {
-    this.dataStream.subscribe(data => this._data = data);
-    this._viewport.renderedRangeStream.subscribe(range => this._onRenderedRangeChange(range));
+    this.dataStream.subscribe(data => {
```

Comment:
Do you ever need to unsubscribe from there?

Reply:
For this one it should be ok, since the underlying Subject is completed in the destroy function, but `renderedRangeStream` I probably should, since that one belongs to the viewport